photo-effects/src/layout/Dashboard.jsx

```diff
@@ -162,12 +240,18 @@ export default class Dashboard extends Component {
   render() {
     return ( 
       <div>
-        <Logout logoutButton={this.logoutButton} />
+        <DashNav />
+        <div className="welcome">
```

Comment:
Please adhere to best practices when formatting your code so its easier for us to read.  Using Prettier extension helps to make this easy.

Reply:
also, the delete buttons arn't working in the projects component.

Reply:
Thanks for the constructive criticism guys i will take that into account :) Hey @claridiva2000 I tested the delete buttons and they were working on my side. Maybe we can take a look at it in a zoom later to see?